galactic/distribution.yaml

```
@@ -1562,6 +1562,19 @@ repositories:
       url: https://github.com/ijnek/nao_interfaces.git
       version: main
     status: developed
+  nao_lola:
+    doc:
+      type: git
+      url: https://github.com/ijnek/nao_lola.git
+      version: main
+    release:
+      tags:
+        release: release/galactic/{package}/{version}
+      url: https://github.com/ijnek/nao_lola-release.git
+      version: 0.0.2-1
+    source:
+      type: git
+      url: https://github.com/ijnek/nao_lola.git
```

It looks like your merge fixup missed lost the branch name:

```suggestion
      url: https://github.com/ijnek/nao_lola.git
      version: main
```

---

Ah, thanks!